adapters/adagio/adagio_test.go

```diff
@@ -0,0 +1,20 @@
+package adagio
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderAdagio, config.Adapter{
+		Endpoint: "http://localhost/prebid_server"})
+
+	if buildErr != nil {
+		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	}
+
+	adapterstest.RunJSONBidderTest(t, "adagiotest", bidder)
```

Comment:
It may not be a bad idea to have a multiformat test. And a test or two to exercise that bad response paths in MakeBids().

Reply:
hi @hhhjort, thanks for your review. I added some tests. Let me know if it is ok you.

Reply:
Is there a reason you hardcoded some tests in by hand rather than adding more JSON tests?